clustering/api/WFLY-9842_CommandDispatcher_Enhancements.adoc

```diff
@@ -0,0 +1,62 @@
+= Title
```

Comment:
This should be replaced with a title for the feature.

Reply:
Done

Reply:
Did you forget to push the change?

Reply:
I did indeed. :)